### ISSUE TITLE
hw08 ready for grading

### ISSUE THREAD

**Issue by Tangjiahui26** [Tangjiahui26/STAT545-hw-Tang-Jiahui#10]
@vincenzocoia @gvdr @ksedivyhaley @JoeyBernhardt @mynamedaike @pgonzaleze @derekcho

[Homework08](https://github.com/Tangjiahui26/STAT545-hw-Tang-Jiahui/tree/master/hw08)
[My Liquor Store Webpage](https://tangjiahui26.shinyapps.io/hw08/)
[ui.R](https://github.com/Tangjiahui26/STAT545-hw-Tang-Jiahui/blob/master/hw08/ui.R)
[server.R](https://github.com/Tangjiahui26/STAT545-hw-Tang-Jiahui/blob/master/hw08/server.R)

**Comment by farihakhan** [comment]
Your app looks great! 
It definitely meets all of the requirements that were asked for this assignment. You definitely went over and beyond for the feature addition. The change in your inputs compared to the class outline is really nice. I especially like that you added a search option on the tables tab, very impressive.

Your readme was very detailed, and it a nice guide for looking at the features in your app. Your r code is very clear and concise, and the formatting of your code helped with user readability.

I don't have any cons to point you, I think you did a great job!

**Comment by arthursunbao** [comment]
Hi Jianhui,

Nice to see your work done and I got to say that you must have put a lot of efforts into the homework because you have finished almost all the requirement the homework requires. Really nice work!

I don't have any issues regarding the work you did because when I reviewed your source code, I found that you did searched a lot of materials to make every function work including the css stylesheet, logo upload, DT package maintenance,  warning issue resolve, plot separation, css under WWW folder, experiment with shiny package, allow user to download dataset and allow user to search for multiple datasets. 

Really good work. 

Congrats!

Jason

**Comment by ksedivyhaley** [comment]
**App uses ui.R and server.R file:** Yes
**Deployed online:** Yes
**Course repo includes README, code:** Yes
**Acknowledge datset source:** Yes
**New app or 3+ added features:** Yes (10 features)
**Reflection:** Yes

**Comments:**

- App looks great!
- You do not need app.R if you are using ui.R and server.R, and can remove the file from your repo.
- When search returns 1 result, text reads “We found 1 results for you .” It would be nice to print “1 result.”
- Nesting of if/else statements used to filter is hard to follow. If you can't simplify the conditionals, use comments.
- Pop-up message is neat coding, but frankly annoying.
- Good discussion of problems encountered and how you solved them.

*Your mark will be distributed later. If you would like more feedback, please feel free to message me on slack.*
